CREDITS

```diff
@@ -165,6 +165,7 @@ of the Coq Proof assistant during the indicated time:
   Nickolai Zeldovich (MIT 2014-2016)
   Théo Zimmermann (ORCID: https://orcid.org/0000-0002-3580-8806,
                    INRIA-PPS then IRIF, 2015-now)
+  Vincent Semeria (2018)
```

Comment:
You are supposed to insert your name in alphabetical order.

Reply:
Done